build/lib/eksd_releases.sh

```diff
@@ -122,7 +122,7 @@ function build::eksd_releases::get_eksd_component_asset_path() {
 
     echo "$yaml" | yq e "$query" -
     
-    set -$oldopt
+    set -$(echo $oldopt | sed 's/c//') # remove -c from options for when this is invoked with bash -c
```

Comment:
Why did we just run into this? I ran into this on some unrelated bash change recently and didnt look close enough into why.

Reply:
I believe it's because we're setting `-c` in our `shell` options [here](https://github.com/aws/eks-anywhere-build-tooling/blob/c6e0fc90e973ccca6c1dc4879f382bb57996ca66/Common.mk#L4-L4). It seems that once `-c` is set you can't reset or unset it in the way you can other flags, which makes sense given its function.

Reply:
That is an odd argument for flags. It really isn't a flag

Reply:
yeah, it's weird 😕

Reply:
For posterity, these were the error messages:
```
/home/prow/go/src/github.com/aws/eks-anywhere-build-tooling/build/lib/eksd_releases.sh: line 37: set: -c: invalid option
set: usage: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
/home/prow/go/src/github.com/aws/eks-anywhere-build-tooling/build/lib/eksd_releases.sh: line 125: set: -c: invalid option
```